server/dao/productAvailability.js

```diff
@@ -42,3 +42,8 @@ exports.setProductAvailability = (
     quantity: quantity,
   });
 };
+
+exports.getProductPrice = (db, productID) => {
+  return db.collection(productsAvailabilityCollectionName).findOne({"productID": {"$eq":productID}}, {"price":1});
+}; 
+
```

Comment:
You don't need this function because we have getProductAvailability which returns the whole availability and so the price too